pandas/core/series.py

```diff
@@ -3100,6 +3100,39 @@ def aggregate(self, func, axis=0, *args, **kwargs):
 
     agg = aggregate
 
+    _transform_doc = dedent("""
+    Examples
+    --------
+    >>> s = pd.Series(range(5))
+    >>> s.transform(lambda x: (x - x.mean()) / x.std())
+    0   -1.264911
+    1   -0.632456
+    2    0.000000
+    3    0.632456
+    4    1.264911
+    dtype: float64
+
+    >>> s.transform([np.sqrt, np.exp])
+           sqrt        exp
+    0  0.000000   1.000000
+    1  1.000000   2.718282
+    2  1.414214   7.389056
+    3  1.732051  20.085537
+    4  2.000000  54.598150
+
+    See also
+    --------
+    pandas.Series.aggregate
+    pandas.Series.apply
+    """)
+
+    @Appender(_transform_doc)
+    @Appender(generic._shared_docs['transform'] % _shared_doc_kwargs)
+    def transform(self, func, axis=0, *args, **kwargs):
```

Comment:
Hmm generally not sure its worth changing actual implementation for docstrings. If this is solely to isolate the various Examples I'd think it preferable to just have one shared Example docstring that covers Series and DataFrame rather than making code changes like this

Reply:
The doc string is currently inherited from NDFrame, but it's not pretty IMO, see [here](http://pandas-docs.github.io/pandas-docs-travis/generated/pandas.Series.transform.html)

The issues are:
* The current doc string discussed NDFrames, not Series, which is confusing,
* the links in the SeeAlso don't work,
* ``Series.transform``'s signature is in master different from the signature for ``Series.agg`` and ``Series.apply`` in that it misses the ``axis`` parameter that the other two have. This may/may not be a problem (I actually couldn't produce a bug caused by this), but the signature should be consistent amongst the three methods, and I agree with the current design for ``agg`` and ``apply`` (i.e. to have an axis parameter).

Thoughts? I can remove this if there's not consensus.

Reply:
I'd say just use substitution for the class name (i.e. Series or DataFrame) and update the See Also links to point to both the Series and DataFrame methods. One of those will obviously be self referencing, but we've done this in other places as well

Reply:
Ok, I see what you mean. I've made a new commit with common examples but the SeeAlso can actually be made correct if Series gets it's own transform method (which is needed for the signature issue to be resolved).